pkg/kwokctl/runtime/binary/cluster.go

```diff
@@ -708,6 +709,32 @@ func (c *Cluster) Logs(ctx context.Context, name string, out io.Writer) error {
 	return nil
 }
 
+// Logs returns the logs of the specified component.
+func (c *Cluster) CollectLogs(ctx context.Context, name string, dir string) error {
+	var errs []error
+
+	logToPathFn := func(name string) error {
+		f, err := runtime.FileOnHost(filepath.Join(dir, name+".log"))
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+		return c.Logs(ctx, name, f)
+	}
+
+	// construct a slice of methods to collect logs
+	fns := []func() error{
```

Comment:
Get the currently existing components from the configuration and moreover add an audit log if it exists.